cloudflare.php

```diff
@@ -79,17 +47,6 @@ function set_default_keys()
     set_protocol_rewrite();
 }
 
-function cloudflare_filter_xss($input)
-{
-    return htmlentities($input, ENT_QUOTES | ENT_HTML5, 'UTF-8');
-}
-
-// Make sure we don't expose any info if called directly
-if (!function_exists('add_action')) {
-    echo "Hi there!  I'm just a plugin, not much I can do when called directly.";
-    exit;
-}
```

Comment:
Currently this is useless since the user can see other files by url like
mydomain.com/wordpress/wp-content/plugins/cloudflare/src/anyPHPFiles.php

I'll need to find a general solution to disable the user to look at any one of them. 

I don't think it's a vulnerability since the code is already on github but...